mirage/factories/donation-goal.js

```diff
@@ -0,0 +1,6 @@
+import { Factory, faker } from 'ember-cli-mirage';
+
+export default Factory.extend({
+  amount: faker.list.random(10000, 20000, 15020),
```

Comment:
Mirage does not support custom attribute transforms, as far as I can tell, so we have to deal with cents. It makes sense, really. Mirage is there to mock the API, and the API deals with cents.